pkg/cloudprovider/gce/gce.go

```diff
@@ -41,6 +41,8 @@ import (
 	"google.golang.org/cloud/compute/metadata"
 )
 
+const EXTERNAL_IP_METADATA_URL = "http://169.254.169.254/computeMetadata/v1/instance/network-interfaces/0/access-configs/0/external-ip"
```

Comment:
It seems like every URL passed into getMetadata will have the same prefix. It looks like this is the first one, but we will want to extract the common prefix out once we are making other requests to the metadata server.